CONTRIBUTING.md

```diff
@@ -0,0 +1,59 @@
+## Contributing In General
+
+Our project welcomes external contributions! If you have an itch, please
+feel free to scratch it.
+
+To contribute code or documentation, please submit a pull request to the [GitHub
+repository](https://github.com/IBM/shopping-list-polymer-pouchdb).
+
+A good way to familiarize yourself with the codebase and contribution process is
+to look for and tackle low-hanging fruit in the [issue
+tracker](https://github.com/IBM/shopping-list-polymer-pouchdb/issues). Before embarking on
+a more ambitious contribution, please quickly [get in touch](#communication)
+with us.
+
+**We appreciate your effort, and want to avoid a situation where a contribution
+requires extensive rework (by you or by us), sits in the queue for a long time,
+or cannot be accepted at all!**
+
+### Proposing new features
+
+If you would like to implement a new feature, please [raise an
+issue](https://github.com/IBM/shopping-list-polymer-pouchdb/issues) before sending a pull
+request so the feature can be discussed. This is to avoid you spending your
+valuable time working on a feature that the project developers are not willing
+to accept into the code base.
+
+### Fixing bugs
+
+If you would like to fix a bug, please [raise an
+issue](https://github.com/IBM/shopping-list-polymer-pouchdb/issues) before sending a pull
+request so it can be discussed. If the fix is trivial or non controversial then
+this is not usually necessary.
+
+### Merge approval
+
+The project maintainers use LGTM (Looks Good To Me) in comments on the code
+review to indicate acceptance. A change requires LGTMs from two of the
+maintainers of each component affected.
+
+For more details, see the [MAINTAINERS](MAINTAINERS.md) page.
+
+## Communication
+
+Please feel free to connect with us: [here](https://github.com/IBM/shopping-list-polymer-pouchdb/issues)
+
+## Setup
```

Comment:
It appears that the "setup," "testing," and "coding style guidelines" sections are still placeholders?

Reply:
We have bad habit of leaving this file as-is, but fixed it now. Mostly refers back to README.